### ISSUE TITLE
Investigate caching in workflows using JPS

### ISSUE THREAD

**Issue by sonakshisaxena1** [android-graphics/intellij-community#16]
Once #13 is closed, investigate caching of artifacts and detection of changes when JPS builds platform using github workflow. This will significantly reduce build and test time.

**Comment by emrekultursay** [comment]
We don't have a lot of PRs in this repo, the action builds only on Linux, and the build takes about 19 minutes. So, let's not spend much time on optimizing this (i.e., low priority).

**Comment by sonakshisaxena1** [comment]
Agreed, just wanted to document this somewhere. This exploration will also be important in designing the build system for the tool.
